dataeng/jobs/analytics/EventExportIncremental.groovy

```diff
@@ -39,7 +39,10 @@ class EventExportIncremental {
                 steps {
                     shell(dslFactory.readFileFromWorkspace('dataeng/resources/event-export-incremental.sh'))
                     if (env_config.get('SNITCH')) {
-                        shell('curl https://nosnch.in/' + allVars.get('SNITCH'))
+                        shell('curl https://nosnch.in/' + env_config.get('SNITCH'))
+                    }
+                    if (env_config.get('OPSGENIE_HEARTBEAT_NAME') && env_config.get('OPSGENIE_HEARTBEAT_KEY')){
```

Comment:
I'm not sure that the HEARTBEAT_KEY will show up in env_config -- I expect it in allVars.

Or you could remove this, since it doesn't have a heartbeat_name.